test/check-application

```diff
@@ -1184,6 +1185,86 @@ class TestApplication(testlib.MachineCase):
               .selectImageAndDownload() \
               .expectDownloadErrorForNonExistingTag()
 
+    def testPullImage(self):
+        b = self.browser
+        m = self.machine
+
+        self.execute(True, f"""
+            podman run -d -p 5000:5000 --name registry --stop-timeout 0 {IMG_REGISTRY}
+            podman run -d -p 6000:5000 --name registry_alt --stop-timeout 0 {IMG_REGISTRY}
+        """)
+
+        # Add local insecure registry into registries conf
+        m.write("/etc/containers/registries.conf", REGISTRIES_CONF)
+        self.execute(True, "systemctl stop podman.service")
+
+        self.execute(True, f"""
+            podman tag {IMG_BUSYBOX} localhost:5000/my-busybox
+            podman push localhost:5000/my-busybox
+            # Remove image with the same sha id
+            podman rmi localhost/test-busybox:latest
+        """)
```

Comment:
This copies the same code from testDownloadImage() which are of course also related. Perhaps either split this out in to a `self.setupLocalRegistries()` helper, or move the new tests into testDownloadImage()?

Reply:
That's also a nice helper, if you need it multiple times.